documentation/modules/managing/con-broker-config-properties.adoc

```diff
@@ -516,5 +555,6 @@ group.initial.rebalance.delay.ms=3000
 # ...
 ----
 
-Essentially, the delay is the amount of time the Kafka group coordinator waits for new consumers to join a group.
-If a longer delay is added, you might experience fewer rebalances, but processing is also delayed.
+When creating or rebalancing a group, the delay is the amount of time that the coordinator waits for members to join. The longer the delay,
+the more likely it is that all the members will join in time and avoid a rebalance.
+But the delay also prevents the group from consuming until the period has ended.
```

Comment:
It's not quite this simple. But it's probably best to do that elsewhere than in the broker configs doc.